packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -391,6 +391,12 @@ class RangeSlider extends StatefulWidget {
   // Touch width for the tap boundary of the slider thumbs.
   static const double _minTouchTargetWidth = kMinInteractiveDimension;
 
+  /// Used to determine if the RenderObject is a _RenderRangeSlider during testing.
+  @visibleForTesting
+  static bool isRangeSliderRenderObject(RenderObject renderObject) {
```

Comment:
Why are these type methods necessary?

Reply:
We should be to find `_RenderRangeSlider` without this, in the tests. Please reference similar tests.